slowish/models.py

```diff
@@ -75,3 +75,21 @@ class Meta:
 
     def __unicode__(self):
         return "{0} (in account {1})".format(self.name, self.account.id)
+
+
+class SlowishFile(models.Model):
+    """A file, (within a particular container)."""
+
+    container = models.ForeignKey(SlowishContainer, on_delete=models.CASCADE)
+
+    path = models.CharField(
+        help_text="Complete path of this file (within the container).",
+        max_length=255)
```

Comment:
Something to consider is that https://docs.djangoproject.com/en/1.8/ref/databases/#character-fields states VARCHARs that have `unique=True` are restricted to 255. The first criteria depends on the DB, but we don't run into the second restriction. It may make sense to make the path unique however in which case I would recommend using 255 as the max length.